### DISCUSSION TITLE
Allow prefix to not be case sensitive

### DISCUSSION THREAD

**Discussion by VariedTechnique** [DiscordDungeons/Bugs#1155] [Ideas]
Discord prefix is case sensitive. I set prefix to D and required a capital D to use commands. Switched prefix to d and it D stopped working

**Comment by FelixArgyle** [comment]
Not a bug, that is intended to happen as your setting it to the exact entry you input.

Bots really aren't for alphabetical prefixes and there likely won't be any additions just to support so prefixes aren't case sensitive for "prefix"

_also in future use the actual template [template here](https://github.com/DiscordDungeons/Bugs/issues/new?assignees=&labels=bug&template=bug_report.yml&title=🐛+Bug+Report%3A+)_

**Comment by Sven65** [comment]
I told them to report this as a bug, with the intention of discussion regarding if we should have case-insensitive prefixes.
